app/controllers/api/v1/movie_person_controller.rb

```diff
@@ -0,0 +1,26 @@
+require 'date'
+class Api::V1::MoviePersonController < ApplicationController
+  def show 
+    api_key = Rails.application.credentials.movie_db[:api_key]
+    response = RestClient.get("https://api.themoviedb.org/3/person/#{params[:id]}/movie_credits?api_key=#{api_key}")
+    response_body = JSON.parse(response.body)
+    reply = serialize_actor(response_body)
+    render json: reply[:data], status: reply[:status]
+  end
+
+  private
+
+  def serialize_actor(data)
```

Comment:
/clap

Reply:
Yes, very good! Are we tracking actors only here, by the way? Since you return `role: "Actor( ... `.

Reply:
The api return from tmdb api divided the return into two separate parts, cast, and crew. They looked slightly different. Currently cast is being returned. Crew had A LOT of irrelevant entries in it, and needs discussion on where to draw the line

Reply:
This has been addressed, we are now also returning actors.